src/Approvals/SFA.DAS.Approvals/Application/Apprentices/Queries/ChangeEmployer/ConfirmEmployer/GetConfirmEmployerQueryHandler.cs

```diff
@@ -33,7 +36,7 @@ public async Task<GetConfirmEmployerQueryResult> Handle(GetConfirmEmployerQuery
             var apprenticeship = apprenticeshipTask.Result;
             var accountLegalEntity = accountLegalEntityTask.Result;
 
-            var isFlexiJobAgency = await IsLegalEntityOnFjaaRegister(accountLegalEntity.MaLegalEntityId);
+            var isFlexiJobAgency = await _fjaaService.IsLegalEntityOnFjaaRegister(accountLegalEntity.MaLegalEntityId);
```

Comment:
I can see the appeal of doing this. However, since we already know the MaLegalEntityId, by calling this method we are doing unnecessary work (retrieving the ale for a second time). Is it worth either a) leaving it unchanged, or b) exposing/splitting the second part of the `IsLegalEntityOnFjaaRegister` where we do the register check as a separate method so that we can call it from here.

To put it another way, should we have this:

```
    public interface IFjaaService
    {
        Task<bool> IsAccountLegalEntityOnFjaaRegister(long accountLegalEntityId);
        Task<bool> IsLegalEntityOnFjaaRegister(long legalEntityId);
    }
```